importer/kmb_massload.py

```diff
@@ -13,6 +14,13 @@
 THROTTLE = 0.5
 LOGFILE = 'kmb_massloading.log'
 
+try:
```

Comment:
A cleaner solution is to just import requests.

Above we would need to change:
*  `from xml.dom.minidom import parse` -> `from xml.dom.minidom import parseString`
* Add `import requests`
Below we would need to change:
* `f = urllib2.urlopen(url)` -> `r = requests.get(url)`
* `urllib2.HTTPError` -> `requests.HTTPError`
* `dom = parse(f)` -> `dom = parseString(f)`